Script/AtomicEditor/editor/Editor.ts

```diff
@@ -103,14 +103,18 @@ class Editor extends Atomic.ScriptObject {
         this.parseArguments();
     }
 
-    initUI() {
-
-      var ui = Atomic.ui;
-      ui.loadSkin("AtomicEditor/resources/default_skin/skin.tb.txt", "AtomicEditor/editor/skin/skin.tb.txt");
-      ui.addFont("AtomicEditor/resources/vera.ttf", "Vera");
-      ui.addFont("AtomicEditor/resources/MesloLGS-Regular.ttf", "Monaco");
-      ui.setDefaultFont("Vera", 12);
-
+    initUI() { 
+        var defSkin = Preferences.getInstance().getApplicationPreference("uiData", "skinPath", "AtomicEditor/editor/skin/");
+        var defDefaultSkin = Preferences.getInstance().getApplicationPreference("uiData", "defaultSkinPath", "AtomicEditor/resources/default_skin/");
+        var defFontFile = Preferences.getInstance().getApplicationPreference("uiData", "fontFile", "AtomicEditor/resources/vera.ttf");
+        var defFontName = Preferences.getInstance().getApplicationPreference("uiData", "fontName", "Vera");
+        var defFontSize = Preferences.getInstance().getApplicationPreference("uiData", "fontSize", 12);
```

Comment:
Could we use UserInterfaceData here instead of Preferences.getInstance().getApplicationPreference calls?  The block is a little dense, and with more prefs will get denser?